api/lib/notifications/notificationController.js

```diff
@@ -0,0 +1,139 @@
+const database = require('../database')
+const config = require('../../config')
+const { eventSubscription } = require('../subscriptions')
+const { getTopic, getMessageTitle, getPushLink } = require('./notifications')
+const { getDefaultEMailSubscriptions } = require('./notifications-types')
+const firebaseAdmin = require('./firebase')
+const Sentry = require('@sentry/node')
+const { UserInputError } = require('apollo-server')
+
+class NotificationController {
+  constructor(networks) {
+    this.db = database(config)
+    this.registrations = {
+      'cosmos1px678cw4m5j5ye52ydjapf2kcw5edn0p6u9cry_transactionSend_cosmos-hub-testnet': {
+        email: { abc: true },
+        push: {}
+      }
+    }
+    // TODO networks might not update here
```

Comment:
This would be a very rare case no? I mean, networks need to change right after you pass them here. Highly unlikely?

Reply:
The networks get passed only add API initialization. So whenever you update them in the db they don't get updated here? (maybe they are because we are actually manipulating the network object)